src/libraries/Microsoft.PowerFx.Core/Public/Types/FormulaType.cs

```diff
@@ -18,8 +18,10 @@ namespace Microsoft.PowerFx.Types
     [ThreadSafeImmutable]
     public abstract class FormulaType
     {
-        // protected isn't enough to let derived classes access this.
-        internal readonly DType _type;
+#pragma warning disable SA1300 // Element should begin with upper-case letter
```

Comment:
Disabled to reduce unrelated noise in this PR, after checking in, I'll fix this.